src/engine/srv.c

```diff
@@ -461,8 +461,7 @@ dss_srv_handler(void *arg)
 		}
 
 		rc = ABT_thread_create(dx->dx_pools[DSS_POOL_NVME_POLL],
-				       dss_nvme_poll_ult, attr,
-				       ABT_THREAD_ATTR_NULL, NULL);
+				       dss_nvme_poll_ult, NULL, attr, NULL);
```

Comment:
Ahhh, I remembered that I changed NVMe poller ULT to use larger stack size for hotplug. (when SPDK hotplug poller is enabled, NVMe poller will run into deeper stack), but I just misplaced the parameter. Thanks for catching this!

BTW, there is a typo in the commit comment, it's poller but not puller. :)